waves/src/index.tsx

```diff
@@ -1,18 +1,18 @@
 import { ChakraProvider } from "@chakra-ui/react";
 import React from "react";
 import ReactDOM from "react-dom";
+import { SSEProvider } from "react-hooks-sse";
 import App from "./App";
-import { Provider as RateServiceProvider } from "./hooks/RateService";
 import "./index.css";
 import reportWebVitals from "./reportWebVitals";
 import theme from "./theme";
 
 ReactDOM.render(
     <React.StrictMode>
         <ChakraProvider theme={theme}>
-            <RateServiceProvider value="http://getbestrate.com">
+            <SSEProvider endpoint="http://localhost:3031/rate/lbtc-lusdt">
```

Comment:
This should be a relative URL otherwise it will not work in production.

Reply:
True, this needs some more investigation. Unfortunately it does not work with relative urls. My assumption is that it's because of the proxy. The server receives the request and keeps sending, but the client does not get the response.

Reply:
ah, it could also because of stuff running in the docker container ... not sure ...